tests/e2e/features/3DViewer.feature

```diff
@@ -5,30 +5,22 @@ Feature: preview 3D model
 
 
     Scenario: preview 3D model
-        Given the user has logged in with username "admin" and password "admin"
-        And the user has uploaded the following 3D models:
+        Given the following 3D models have been uploaded:
+            | filename   |
             | model1.glb |
             | model2.glb |
+        And the user has logged in with username "admin" and password "admin"
 
         # preview 3D model
         When the user previews the file "model1.glb" in the 3D model viewer
         Then the 3D model "model1.glb" should be displayed in the viewport
         And the file name "model1.glb" should be shown in the topbar
 
-        # toggle between standard and fullscreen mode
-        When the user enters fullscreen mode
-        Then the 3D model should be displayed in fullscreen mode
-        And the topbar should not be visible
-        When the user exits fullscreen mode
-        Then the 3D model should be display in standard mode
-        And the file name "model1.glb" should be shown in the topbar
-
-        # rotate, zoom and reset 3D model
-        When the user rotates the model
-        And the user zooms into the model
```

Comment:
may be we can use snapshot like this blog suggest for those https://www.chromatic.com/blog/how-to-visual-test-ui-using-playwright/

Reply:
yeah, implementing the zoom and rotation interaction of the object is fairly easy, the assertion is the challenge I wasn't able to solve so far... we decided to do that in a separate PR. For this reason I've deleted the corresponding steps in the feature and also the implementation of these steps in the ViewerContext for now. I'm adding your comment to the issue for that.